validation/src/main/scala/hmda/validation/rules/lar/validity/V485.scala

```diff
@@ -0,0 +1,31 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Predicate, Result }
+import hmda.validation.rules.EditCheck
+import scala.util.Try
+
+object V485 extends EditCheck[LoanApplicationRegister] {
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    val applicant = lar.applicant
+    when(applicant.coRace1 is containedIn(1 to 5)) {
+      (applicant.coRace2 is validRace) and
+        (applicant.coRace3 is validRace) and
+        (applicant.coRace4 is validRace) and
+        (applicant.coRace5 is validRace)
+    }
```

Comment:
If you'd like, you can do something like `List(applicant.coRace2, applicant.coRace3, applicant.coRace4, applicant.coRace5).map(_ is validRace).reduce(_ and _)` to remove some of the duplication.  Does it lose readability? I can't tell at the moment.

Likewise in V470: `List(applicant.race2, applicant.race3, applicant.race4, applicant.race5).forall(validRace(_)) is equalTo(true)`

I don't know whether @jmarin's changes will make one version make more sense than the other.